crates/flycheck/src/lib.rs

```diff
@@ -50,13 +50,15 @@ pub enum FlycheckConfig {
         extra_args: Vec<String>,
         extra_env: FxHashMap<String, String>,
         ansi_color_output: bool,
+        target_dir: Option<PathBuf>,
     },
     CustomCommand {
         command: String,
         args: Vec<String>,
         extra_env: FxHashMap<String, String>,
         invocation_strategy: InvocationStrategy,
         invocation_location: InvocationLocation,
+        target_dir: Option<PathBuf>,
```

Comment:
```suggestion
```

If the user uses a custom command they should be specifying the target dir themselves in the args as we can't predict what format their command expect.

This is the bahvior with all cargo config args, we drop them all for the custom command